dev/merge_arrow_pr.py

```diff
@@ -460,24 +461,48 @@ def get_version_json(version_str):
     return [get_version_json(v) for v in issue_fix_versions]
 
 
-def connect_jira(cmd):
-    # ASF JIRA username
-    jira_username = os.environ.get("APACHE_JIRA_USERNAME")
+CONFIG_FILE = "~/.config/arrow/merge.conf"
+
+
+def load_configuration():
+    config = configparser.ConfigParser()
+    config.read(os.path.expanduser(CONFIG_FILE))
+    return config
+
+
+def get_credentials(cmd):
+    username, password = None, None
+
+    config = load_configuration()
+    if "jira" in config.sections():
+        username = config["jira"].get("username")
+        password = config["jira"].get("password")
 
-    # ASF JIRA password
-    jira_password = os.environ.get("APACHE_JIRA_PASSWORD")
+    # Fallback to environment variables
+    if not username or not password:
+        username = os.environ.get("APACHE_JIRA_USERNAME")
+        password = os.environ.get("APACHE_JIRA_PASSWORD")
 
-    if not jira_username:
-        jira_username = cmd.prompt("Env APACHE_JIRA_USERNAME not set, "
-                                   "please enter your JIRA username:")
+    # Fallback to user tty prompt
+    if not username or not password:
```

Comment:
Are we sure those are always set _both_? Should we not have:
```python
if not username:
    username = cmd.prompt(...)
if not password:
    password = cmd.getpass(...)
```

Reply:
That would be weird to set one and not the other, but we could.

Reply:
Someone paranoid may not like to store a password in a conf file or an environment variable.

Reply:
We may as well make this change. I tested the script so we can merge this one this is addressed